deepinv/models/wavdict.py

```diff
@@ -377,7 +377,7 @@ def forward(self, x, ths=0.1, **kwargs):
         r"""
         Run the model on a noisy image.
 
-        :param torch.Tensor x: noisy image.
+        :param torch.Tensor x: noisy image. Assumes a tensor of shape (B, C, H, W) (2D data) or (B, C, D, H, W) (3D data).
```

Comment:
am I right in thinking that this denoiser needs no code modification for 3D data?